### PR TITLE
72 liferay upgrade plan doc

### PR DESCRIPTION
**Pull request by jhinkey**
Hi Greg,

This branch contains the plan documents only; all of the actual upgrade articles are in this other branch:

https://github.com/jhinkey/liferay-docs/tree/72-upgrading-liferay/deployment/articles/05-upgrading-to-liferay-7-2

I did that so I can get the upgrade docs reviewed  and published. 

The plans refer to the article blobs that are currently on my GitHub branch (`72-upgrading-liferay`) mentioned above. However, I'll be scrapping this branch as I'm sending the articles to `master` individually in PRs to Rich for review. I need to separate these plans from the articles as it's preventing me from getting the articles reviewed and merged into master---I don't want to publish the plans yet.

The articles have been sent to Rich in PRs #4046-#4055. See https://github.com/sez11a/liferay-docs/pulls/jhinkey.

https://issues.liferay.com/browse/LRDOCS-6301

cc/ @achaparro, @jtydhr88

### CODE REVIEW

File: deployment/articles/100-reference/07-liferay-code-upgrade-plan.markdown [hunk 1]
```diff
@@ -0,0 +1,245 @@
+# Liferay Code Upgrade Plan
```
Comment (gamerson) [inline deployment/articles/100-reference/07-liferay-code-upgrade-plan.markdown:1]: Should this be `Liferay Plugins Upgrade Plan`?

Comment (gamerson) [reply deployment/articles/100-reference/07-liferay-code-upgrade-plan.markdown:1]: Thoughts @jhinkey @jtydhr88

Comment (jtydhr88) [reply deployment/articles/100-reference/07-liferay-code-upgrade-plan.markdown:1]: we also have Liferay Workspace code support, I don't think **Plugins** will cover it.

**Comment by jhinkey** [comment]
@gamerson , @jtydhr88 - I pushed a commit that fills in the descriptions converts the Markdown to HTML.

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**